arquillian-chameleon-container-model/src/main/resources/chameleon/default/containers.yaml

```diff
@@ -147,7 +147,7 @@
   defaultProtocol: Servlet 3.0
   exclude: *EAP_EXCLUDE
 - name: JBoss AS
-  versionExpression: 7.*
+  versionExpression: 7\.[0-1]\.[0-2]\.(.*)$
```

Comment:
yeah I did this way as we want to support for `7.0.2`. I don't see any other option in regex which will select later text depending on first selected text.